### PR TITLE
Fixed #25156: Added mention of django.setup() to the settings overview

### PR DESCRIPTION
**Pull request by ubernostrum**

### CODE REVIEW

File: docs/topics/settings.txt [hunk 1]
```diff
@@ -271,3 +271,29 @@ It boils down to this: Use exactly one of either ``configure()`` or
 
     :doc:`The Settings Reference </ref/settings>`
         Contains the complete list of core and contrib app settings.
+
+Calling django.setup() is required for "standalone" Django usage
```
Comment (timgraham) [inline docs/topics/settings.txt]: add backticks around django.setup()

File: docs/topics/settings.txt [hunk 2]
```diff
@@ -271,3 +271,29 @@ It boils down to this: Use exactly one of either ``configure()`` or
 
     :doc:`The Settings Reference </ref/settings>`
```
Comment (timgraham) [inline docs/topics/settings.txt]: I'd leave the "See also" box at the bottom of the page, so just move your section about it. While you're there it looks like the "login_required" link is unused.

File: docs/topics/settings.txt [hunk 3]
```diff
@@ -271,3 +271,29 @@ It boils down to this: Use exactly one of either ``configure()`` or
...
         Contains the complete list of core and contrib app settings.
+
+Calling django.setup() is required for "standalone" Django usage
+----------------------------------------------------------------
+
+If you're using components of Django "standalone" -- for example,
```
Comment (timgraham) [inline docs/topics/settings.txt]: Please make lines longer (wrap at 79 chars).

**Comment by timgraham** [comment]
Content looks good and please make your commit message in past tense.

**Comment by ubernostrum** [comment]
@timgraham done on all three counts.

**Comment by timgraham** [comment]
merged in 5281f8b635f10b2839dc146d7e44f7d94d1955d8, thanks!

**Pull request closed without merging**